internal/driver/interactive.go

```diff
@@ -82,9 +95,15 @@ func interactive(p *profile.Profile, o *plugin.Options) error {
 					continue
 				}
 				// Allow group=variable syntax by converting into variable="".
-				if v := pprofVariables[value]; v != nil && v.group == name {
-					if err := pprofVariables.set(value, ""); err != nil {
-						o.UI.PrintErr(err)
+				if okValues := groups[name]; okValues != nil {
+					if v := pprofVariables[value]; v != nil && v.group == name {
+						if err := pprofVariables.set(value, ""); err != nil {
+							o.UI.PrintErr(err)
+						}
+					} else {
+						sort.Strings(okValues)
+						recognizedValues := strings.Join(okValues, ", ")
+						o.UI.PrintErr(fmt.Errorf("Unrecognized value for %s: %q. Recognized values are %s", name, value, recognizedValues))
```

Comment:
". Recognized values are" -> ", use one of", to avoid repeating longish "recognized" stem in the sentence.

Reply:
Done.